### PR TITLE
Fixes after first audit

### PR DESCRIPTION
**Pull request by danieljcss**
Minor improvements, accepted all the recommendations.

### CODE REVIEW

File: contracts/GrizzlyVault.sol [hunk 1]
```diff
@@ -323,6 +322,8 @@ contract GrizzlyVault is IUniswapV3MintCallback, IUniswapV3SwapCallback, Grizzly
 			uint128 newLiquidity = _liquidityForAmounts(ticks, sqrtRatioX96, amount0Max, amount1Max);
 
 			mintAmount = uint256(newLiquidity);
+			require(mintAmount > MIN_INITIAL_SHARES, "min shares");
```
Comment (borus-dev) [inline contracts/GrizzlyVault.sol:326]: The require will make the getter function revert
In that case we would need to add a try catch when using that function in the Zap

It is more appropriate to just return, without revert `return(0, 0, 0)`

File: contracts/GrizzlyVaultFactory.sol [hunk 2]
```diff
@@ -44,8 +44,11 @@ contract GrizzlyVaultFactory is IGrizzlyVaultFactory, Ownable {
 	function getVaults(address deployer) public view returns (address[] memory) {
```
Comment (borus-dev) [inline contracts/GrizzlyVaultFactory.sol:44]: Not sure what effect in gas saving has in a getter function that is meant to be used externally, but not from smart contracts, nevertheless no harm in leaving like that

File: contracts/GrizzlyVault.sol (hunk 1 above)
Comment (danieljcss) [reply contracts/GrizzlyVault.sol]: I agree, committed the suggested change

File: contracts/GrizzlyVaultFactory.sol (hunk 2 above)
Comment (danieljcss) [reply contracts/GrizzlyVaultFactory.sol:44]: We can leave it like that in case of some external contract integrates the Zap in a non-view function

**Comment by borus-dev** [comment]
Reviewed

**Pull request merged**